apix-impl/src/integration-test/java/eu/xenit/apix/tests/BaseTest.java

```diff
@@ -87,6 +90,20 @@ protected boolean removeTestNode(NodeRef nodeRef) {
         return success;
     }
 
+    protected File createTextFileWithContent(String fileName, String content) throws IOException {
+        if (!fileName.endsWith(".txt")) {
+            fileName += ".txt";
+        }
+        File textFile = new File(fileName);
+        textFile.deleteOnExit();
+        textFile.createNewFile();
+        PrintWriter writer = new PrintWriter(fileName, "UTF-8");
+        String contentString = content;
+        writer.println(contentString);
+        writer.close();
```

Comment:
How about https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html instead of `close` to make it safer?